### ISSUE TITLE
Branding page

### ISSUE THREAD

**Issue by emileber** [saglacio/saglac.io-original#134]
Il faudrait ajouter une page avec les informations de branding, genre logo, icon, image facebook, etc.

Pages d'exemple pour utilisation du logo:
https://slack.com/brand-guidelines
https://en.facebookbrand.com/guidelines/brand
https://www.patreon.com/brand/guidelines
https://about.twitter.com/en_us/company/brand-resources.html
etc. :)

**Comment by paaudet** [comment]
Si vous décidez de rendre disponible le logo fait par Jessica, je me propose pour en faire une version .ai ou .eps, c'est plus pratique pour l'utilisation courante. Disons que vous virez ça en t-shirt/casquette ou un gros flag pour l'édifice CGI. ;)

**Comment by emileber** [comment]
Il est déjà ailleurs dans une issue, mais je ne me souviens plus laquelle, alors je vais le réuploadé ici.
[logo.zip](https://github.com/saglacio/saglac.io/files/1438126/logo.zip)

**Comment by GabLeRoux** [comment]
J'ai edit la description pour ajouter des liens vers des exemples ;)

**Comment by paaudet** [comment]
J'ai déjà downloadé ce .zip, je travaille avec ça pour les favicon. :)

**Comment by emileber** [comment]
@paaudet oh d'accord! haha, sinon, pour les source, je peux regarder avec Jess mais je crois que c'est tout ce qu'on avait.

Le font c'est Source Code Pro me semble.

**Comment by paaudet** [comment]
@emileber Nha c'est correct, je m'occupe du reste! ^^

**Comment by orditeck** [comment]
@emileber la font est Code Pro Light, je l'ai ajouté dans le drive en version vectorisé afin que tout le monde puisse l'utiliser sans avoir besoin de la font: https://drive.google.com/open?id=1DU-TERq1q1bcBaXA4cWHyhbPy2aRA9ie

C'est une font payante, ça pourrait être intéressant de passer à une font libre de droit éventuellement.

**Comment by emileber** [comment]
Me semble que source code pro est gratuit sur google fonts? non? https://fonts.google.com/specimen/Source+Code+Pro

**Comment by GabLeRoux** [comment]
> Source Code Pro can be adapted and redistributed according to the terms of the Open Font License (OFL) agreement.

**Comment by orditeck** [comment]
Oui, Source Code Pro est gratuite. Mais la font du logo est **Code Pro Light**.

**Comment by orditeck** [comment]
J'ai ouvert une issue en ce sens, https://github.com/saglacio/saglac.io/issues/147

**Comment by GabLeRoux** [comment]
Pam Pam Pam 😱

**Comment by emileber** [comment]
Effectivement, Jess l'a même dit ici: https://github.com/saglacio/saglac.io/issues/77#issuecomment-276826933

**Comment by orditeck** [comment]
Oui, et elle y inclut également la font dans le zip. Je sais pas côté légalité. Faudrait peut-être mettre ça down. Dans tous les cas, le logo est vectorisé dans le Drive donc en théorie nous n'avons plus besoin de la font, donc tant que la personne qui a fait le logo a la license c'est all good I guess ?

Edit : je propose qu'on continue la discussion dans #147

**Comment by emileber** [comment]
Faut juste pas l'utilisé pour autre chose. En théorie, c'est correct qu'elle s'en soit servi pour un logo puisque la license de base sert à ça.
